collectors/product_definitions/core.py

```diff
@@ -115,8 +125,8 @@ def sync_ps_products_modules(ps_products_data: dict, ps_modules_data: dict):
         }
         related_ps_modules = filtered_product_data.pop("ps_modules")
 
-        ps_product, _ = PsProduct.objects.update_or_create(
-            short_name=product_short_name, defaults=filtered_product_data
+        ps_product = PsProduct.objects.create(
+            short_name=product_short_name, **filtered_product_data
         )
```

Comment:
This has been here for quite some time but I never actually realize that having it like this won't erase anything no longer relevant :smile: Good job.